src/pow.cpp

```diff
@@ -73,22 +73,3 @@ unsigned int CalculateNextWorkRequired(const CBlockIndex* pindexLast, int64_t nF
 
     return bnNew.GetCompact();
 }
```

Comment:
Ultimately I agree, however it means we need either to put CBlockIndex as part of consensuslib or making a stripped type representing it in consensuslib.

But it involves lots of refactoring that I don't need it for my VerifyBlock / VerifyTx so I won't do it for now. I'll maybe include nextWorkRequired as field in CConsensusContextInfo though.